includes/models/model.llms.quiz.php

```diff
@@ -108,6 +108,26 @@ public function get_questions( $return = 'questions' ) {
 		return $this->questions()->get_questions( $return );
 	}
 
+	/**
+	 * Get questions count.
+	 *
+	 * @since [version]
+	 *
+	 * @return int Question Count.
+	 */
+	public function get_questions_count() {
+
+		/**
+		 * Filter the count of questions in a quiz.
+		 *
+		 * @since [version]
+		 *
+		 * @param int       $questions_count Number of questions in a quiz.
+		 * @param LLMS_Quiz $quiz            Current quiz object.
+		 */
+		return apply_filters( 'llms_quiz_attempt_questions_count', count( $this->get_questions( 'ids' ) ), $this );
```

Comment:
```suggestion
		return apply_filters( 'llms_quiz_questions_count', count( $this->get_questions( 'ids' ) ), $this );
```
there is no attempt here... of course

Reply:
fixed!